packages/terra-grid/src/GridColumn.jsx

```diff
@@ -27,9 +27,13 @@ const propTypes = {
   /**
    * The number of columns across all breakpoints. From 1-12(inclusively).
    */
-  col: columnRange,
+  tiny: columnRange,
+  /**
+   * The number of columns starts from enormous breakpoint. From 1-12(inclusively).
```

Comment:
Can we update and change these prop descriptions with this breaking change? I find them to be confusing and always have to guess and check my implementation when trying to use them

Reply:
Ya... I wasn't truthfully sure what it meant either. Any clarification as to the significance of the numbers would be appreciated.

Reply:
As you resize thew width of your browser, the grid should re-adjust it's layout when a new breakpoint is reach. This layout will reflect column values provided to the props. I'd update an example and test it out a bit to find a description that makes the best sense to you

Reply:
I'd recommend the following for the prop description.
```
Defines the number of columns this container will span in the 12 column grid. It is applied at the `tiny` breakpoint and wider screens.
```